src/components/header/index.tsx

```diff
@@ -2,25 +2,39 @@ import Image from "next/image";
 import styles from "./styles.module.css";
 import { Navbar } from "./navbar";
 import { SocialMedias } from "../social-medias";
+import Link from "next/link";
 
 export const Header = () => {
   return (
     <header className={styles.header}>
       <div className={styles.header__logo_container}>
-        <figure className={styles.header__logo}>
+        <Link
+          href={"/"}
+          className={styles.header__logo}
+        >
           <Image
             src="/logo/logo.svg"
             alt="logo"
             width={148}
             height={34}
           />
-        </figure>
-        <button className={styles.header__button}>Acessar materiais</button>
+        </Link>
+        <Link
+          href="/materials"
```

Comment:
também lembra que a url tem que ser amigável, em português. o código a gente faz em inglês mesmo, mas o que for ficar acessível pro público é em português, tipo a url

Reply:
alterado: [c2701d6](https://github.com/tech-pro-bem/abre/pull/43/commits/c2701d6b9b5db7637cf34532e476f2ae86e806c2)